static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -45,7 +45,7 @@ const fillAdvertSlots = (
     start: () => void,
     stop: () => void
 ): Promise<void> => {
-    if (commercialFeatures.dfpAdvertising) {
+    if (commercialFeatures.dfpAdvertising || commercialFeatures.adFree) {
```

Comment:
Makes sense... We have to remember that when we switch off advertising in the switchboard, it doesn't switch off ad free. This means that if something happens to break in here we need to be aware that it doesn't stop the DFP and the Sonobi code from running (and potentially other programmatic partners)